deepmd/pt/optimizer/hybrid_muon.py

```
@@ -106,13 +120,31 @@
 try:
     import triton
     import triton.language as tl
 
-    TRITON_AVAILABLE = True
+    try:
+        # Touching ``driver.active`` forces the lazy proxy to initialize the
+        # backend driver. ``get_current_target`` is the lightest public call
+        # that exercises the same path as ``Autotuner.__init__``.
+        triton.runtime.driver.active.get_current_target()
+        TRITON_AVAILABLE = True
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)